lib/openc/json_schema/date_converter.rb

```diff
@@ -13,7 +13,7 @@ def _convert_dates(record, validator, json_schema, schema)
         return record if schema.nil?
 
         if (ref = schema['$ref'])
-          schema_uri = validator.absolutize_ref_uri(ref, json_schema.uri)
+          schema_uri = JSON::Util::URI.absolutize_ref(ref, json_schema.uri)
```

Comment:
Wonder if `validator` should be removed from the code then, seeing that it's not used.

Reply:
Like 6b662b1d43cf94ad4bc5818ef501f8614365f0ad ?

Reply:
For posterity, have to change this method because it was removed as duplicate in https://github.com/ruby-json-schema/json-schema/commit/eccf80f492462b34afd8ddd6b7e0509231d069b7